code/items/equipments/TTTEquipment.cs

```diff
@@ -16,20 +16,20 @@ public EquipmentAttribute(string name) : base(name)
 
     public abstract class TTTEquipment : Networked, IBuyableItem
     {
-        private string name { get; set; }
+        private string Name { get; set; }
```

Comment:
All properties are pascal case regardless of access modifier, not sure about the pro vs. cons about handling it this way.